src/main/java/com/rultor/profiles/ProfileDeprecationAware.java

```diff
@@ -0,0 +1,151 @@
+import com.jcabi.xml.XML;
+import com.rultor.spi.Profile;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Decorator allowing to add a deprecation message if and only if the
+ * {@link Profile} is deprecated.
+ * @author Nicolas Filotto (nicolas.filotto@gmail.com)
+ * @version $Id$
+ * @since 1.62
+ */
+public final class ProfileDeprecationAware implements Profile {
+
+    /**
+     * The underlying profile.
+     */
+    private final transient Profile profile;
+
+    /**
+     * Constructs a {@code ProfileDeprecationAware} with the specified
+     * underlying {@code Profile}.
+     * @param prof The underlying profile
+     */
+    public ProfileDeprecationAware(final Profile prof) {
+        this.profile = prof;
+    }
+
+    @Override
+    public String name() {
+        return this.profile.name();
+    }
+
+    @Override
+    public XML read() throws IOException {
+        final XML xml = this.profile.read();
+        this.check(xml);
+        return xml;
+    }
+
+    @Override
+    public Map<String, InputStream> assets() throws IOException {
+        return this.profile.assets();
+    }
+
+    /**
+     * Checks if the profile is deprecated and if so log the deprecation notice.
+     * @throws IOException if it fails while getting the XML format of the
+     *  profile
+     */
+    public void check() throws IOException {
+        this.check(this.profile.read());
+    }
+
+    /**
+     * Indicates whether the profile is deprecated or not.
+     * @return True if the profile is deprecated, false otherwise
+     * @throws IOException if it fails while getting the XML format of the
+     *  profile
+     */
+    public boolean deprecated() throws IOException {
+        return ProfileDeprecationAware.deprecated(this.profile.read());
+    }
+
+    /**
+     * Checks if the profile is deprecated and if so log the deprecation notice.
+     * @param prof The XML representation of the profile to check
+     */
+    private void check(final XML prof) {
+        if (ProfileDeprecationAware.deprecated(prof)) {
+            Logger.warn(this, "#### Deprecation Notice ####");
+            Logger.warn(
+                this,
+                "You are using the Rultor default Docker image in your build."
+            );
+            Logger.warn(this, "The Rultor has to:");
+            Logger.warn(
+                this,
+                "1. Provide the sudo package/command and not stop doing so "
+            );
+            Logger.warn(
+                this,
+                "whenever a change to the Dockerfile is made, even if Rultor "
+            );
+            Logger.warn(this, "itself does not need the sudo command.");
+            Logger.warn(
+                this,
+                "2. Not install any gems to the global scope that interfere "
+            );
+            Logger.warn(this, "with pdd or est");
+            Logger.warn(this, "########################");
```

Comment:
@essobedo in Netbout under `http://www.netbout.com/b/xxxxxx?open=agenda` you have a list of the tasks assigned to you. This one should have `1 impediment` listed in the details column.
This should be a clear indicator for you, that you should at least make sure that nothing is blocking your from a proper solution. Better yet if this is at all ambiguous just ping me and I'll look into it asap :)

At any rate, [this is a build output](http://www.rultor.com/t/7608-195025457), are you sure that log messages from `LOGGER` go into that thing? ( I'm not tbh).
This `LOGGER` is Rultor's internal logger, running somewhere on Heroku, why would they be appended to the output of the build, that is a stream from the Docker container?